src/non-standalone/SignaturePad/SignaturePad.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import SignPad from "../../standalone/SignPad/index";
+import SignPadDialog from "../Dialog/SignPadDialog";
+
+export interface SignaturePadCanvasProps {
```

Comment:
don't copy paste all of these. use `interface Name extends OtherInterface, AnotherInterface` instead. You can also use `Omit<T, Keys>` and `Pick<T, Keys>` to import only a part of an interface